yarn.lock

```diff
@@ -2391,9 +2391,15 @@ domhandler@2.3:
   dependencies:
     domelementtype "1"
 
+<<<<<<< HEAD
```

Comment:
Actually never mind the yarn version thing, looks like you’ve just got some merge conflict in here (I usually just check out `yarn.lock` from master, then re-run `yarn` to generate the right one in this situation).